### DISCUSSION TITLE
Changing resolution of game causes screen to go black and computer to be unresponsive but playing game audio

### DISCUSSION THREAD

**Discussion by RobertLinsalato** [ppy/osu#17993] [Q&A] [answered]
### Type

Other

### Bug description

Going through all the settings for _osu! Lazer_ and I wanted to test another resolution. Changed from 1920x1080 to 1260x720, and my laptop proceeded to go black and be completely unresponsive. All the audio in game was still playing, but peripherals and their respective programs like _Corsair ICUE_ and _Logitech G Hub_ we're frozen and also unresponsive. To fix the problem, I had to hard reset my laptop by holding the power button.

### Version

2022.424.1-lazer

### Logs

N/A, logs were lost when I had to restart my laptop.

**Comment by peppy** [comment, marked as answer]
This generally means your display could not handle the resolution.

You can hit alt-enter or alt-f4 to avoid rebooting.

**Reply by RobertLinsalato** [reply]
Not trying to be condescending, but having a 30 Series GPU and a 6th Gen CPU didn't seem to help. When it happened, I did both `alt-enter` and `alt-f4` but neither seemed to fix the problem. What confused me is that it _**wasn't**_ going up in resolution, but I was turning the resolution **_down_**. Thank you for the response though :)

**Reply by peppy** [reply]
It's not related to CPU or GPU, it's more about what the monitor can handle.

When this happens, can you open your monitor OSD and take a photo or note down the details it is showing?

**Reply by RobertLinsalato** [reply]
I've tried to recreate it a few times, but it works fine now. If it happens again, I will put a picture in this discussion. This is all running on a laptop btw, could that be a reason it happened?

**Reply by peppy** [reply]
It's hard to say. A case like this usually means there's an issue at a driver level (if we assume that drivers are not supposed to get into a bad state regardless of what the application is doing).

**Reply by RobertLinsalato** [reply]
Ok, just checked and my drivers are up to date. Will look out for the problem if it comes back. Thanks for the help!
